File: Second.cc
#include <iostream>
using namespace std;
int main (){
    cout <<"This Code Is Used To Compute For The Net Pay Of Employees In A Particular Company"<<endl;
    double grossPay , netPay , incomeTax, nationalLevy, districtTax, payMent;
    // Starting the Main Code 
    int hours, children;
    string gender ;
    cout<<"Please enter the gender of ther person"<<endl;
    cin>>gender;
    cout<<"Please, enter the hours worked by the person:";
    cin>>hours;
    //Writing a nexted if statement 
    if (hours<=40) {
        cout<<"It's regular Hours"<<endl;
        if(gender =="male") {
            grossPay = 500*hours;
        } else {
            grossPay = 550*hours;
        }
    } else {
        cout<<"It is an overtime"<<endl;
        if (gender =="male") {
            grossPay = 1.5*500*hours;
        } else {
            grossPay = 1.5*550*hours;
        }
    }
    //Writing a code for the number of children
    cout<<"Please, enter the number of children"<<endl;
    cin>>children;
    if (children<=3) {
        payMent = children*10;
    } else {
        payMent = children*10 +(children-3)*20;
    }
    incomeTax = 0.15*grossPay;
    nationalLevy = 0.025*grossPay;
    districtTax = 0.01*grossPay;
    netPay = grossPay -(incomeTax+nationalLevy+districtTax+payMent);
    cout<<netPay;

    return 0;
}

File: Second.java
import java.util.Scanner;
public class Second {
    public static void main(String[] args) {
        Scanner inputScanner = new Scanner (System.in);
        System.out.println("This code Is Used To Compute For The Net Pay Of Employees In A Particular Company.\n");
        double grossPay , netPay, incomeTax, nationlLevy, districtTax,payMent;
        //Starting The Main Code
        System.out.println("Please enter the gender of the person:");
        String gender = inputScanner.nextLine();
        System.out.println("Please , enter the hours worked by the person: ");
        int hours = inputScanner.nextInt();
        //Starting A while Loop
        if (hours<=40){
            System.out.println("It's regular Hours");
            if (gender.equals("male")) {
                grossPay = 500*hours;
            }else{
                grossPay = 550*hours;
            }
            }else{
                System.out.println ("It is an over time");
                if(gender == "male") {
                    grossPay = 1.5*500*hours;
                } else{
                    grossPay = 1.5*550*hours;
                }
            }
            System.out.println("Please enter the number of children");
            int children = inputScanner.nextInt();
            if (children<=3) {
                payMent = children*10;
            } else {
                payMent = children*10 +(children-3)*20;
            }
            incomeTax = 0.15*grossPay;
            nationlLevy = 0.025*grossPay;
            districtTax = 0.01*grossPay;
            netPay = grossPay - (incomeTax+nationlLevy+districtTax +payMent);

            System.out.println("The net pay of the employee is: " +netPay);

            inputScanner.close();
        }
        
    }
